CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
```

Comment:
2020 now ;)

Reply:
This is the concern of a different PR :)